Automattic/jetpack
# Add Social Logos to monorepo

Moves Social Logos package into the monorepo

## Proposed changes:

This PR moves the Social Logos package into the monorepo. Originally I was going to wait for some feedback on a related project, but figured it's best to just wrap this up and iterate as needed.

Related tasks:
- [x] Rewrite/clean up readme and any other such files
- [ ] Audit existing issues in the original repo
- [ ] Finish configuring the mirror repo
- [x] Confirm what exactly needs to be committed to the repo and NPM. For example, historically we've only provided `.js` files to NPM, and this now only has the `.jsx` files; I'm not sure if that will be a problem or not. If we need to transpile prior to publishing, maybe [this](https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/webpack-config/README.md) is what I need to understand.
- [ ] Do an official release of the package (v3.0)
- [ ] Maybe make a composer package for the PHP files?

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Run `jetpack build js-packages/social-logos` and verify the `build` folder is generated.
2. Add an SVG to `/src/svg`.
3. Run `./tools/build`
4. Verify the generated files match what one would expect.
